testsuite/features/core/proxy_branch_network.feature

```diff
@@ -21,6 +21,13 @@ Feature: Setup SUSE Manager for Retail branch network
     And I manually install the "bind" formula on the server
     And I synchronize all Salt dynamic modules on "proxy"
 
+@proxy
+@private_net
+  # WORKAROUND
+  Scenario: Remove dhcp packages on the proxy
```

Comment:
I would personally do this scenario as the first scenario, before the installation of formulas, as it is more like a pre-requisite.

Reply:
No problem with that, moving :-)